prog/setup_spdk.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class Prog::SetupSpdk < Prog::Base
+  subject_is :sshable, :vm_host
+
+  def start
+    fail "Not enough hugepages" unless vm_host.total_hugepages_1g > vm_host.used_hugepages_1g
+    sshable.cmd("sudo bin/setup-spdk")
+    sshable.cmd("sudo systemctl start spdk")
+    vm_host.update(used_hugepages_1g: vm_host.used_hugepages_1g + 1)
```

Comment:
We also update `used_hugepages_1g` while creating vm. Some database contstraints similar to used cores might helps to keep values consistent.
https://github.com/ubicloud/ubicloud/blob/dea57e31a323fb6e66287b0012dc438e38b03fe9/migrate/008_add_vm_allocation_counters.rb#L19

Reply:
Great point. I will do this in the next storage PR which changes the schema to reduce number of migrations.